lib/Quantization/Quantization.cpp

```diff
@@ -507,11 +515,37 @@ class FunctionQuantizer : public FunctionConverter {
       // [         RowwiseQuantizedFullyConnectedNode            ]
       //                              |
       //                       [DequantizeNode]
+      bool foundFC = false;
+      NodeValue input, weights, bias, result;
       if (auto *fcN = llvm::dyn_cast<FullyConnectedNode>(Q->getInput())) {
-        NodeValue input = fcN->getInput();
-        NodeValue weights = fcN->getWeights();
-        NodeValue bias = fcN->getBias();
-        NodeValue result = fcN->getResult();
+        foundFC = true;
+        input = fcN->getInput();
+        weights = fcN->getWeights();
+        bias = fcN->getBias();
+        result = fcN->getResult();
+      } else if (const auto *baN =
+                     llvm::dyn_cast<BatchedAddNode>(Q->getInput())) {
```

Comment:
Since now BatchedAddNode could be replaced as well, the comments in later replacement (line 561-562) need to be updated.

Reply:
Also, one more question: if the batchedAdd and Matmul is not lowered from FC (that is, the module has 2 layers as matmal + bachedAdd), do we need to do this conversion?

Reply:
That means that the original model we load would have MatMul + BatchedAdd instead of FC. I'm not sure if we want to do that in this case -- we could, since it is equivalent after all. But that feels like outside of the scope of this PR. My initial intuition for supporting that would be that we'd want an earlier pass to replace MM + BA patterns with FC.

Reply:
>Since now BatchedAddNode could be replaced as well, the comments in later replacement (line 561-562) need to be updated.

Updated the comments.